polaris-react/src/components/TopBar/_variables.scss

```diff
@@ -9,6 +9,6 @@ $search-max-width: 580px;
 // `page-content-when-not-partially-condensed`
 // Once the frame is rendering our React routes
 $large-width: calc(
-  #{layout-width(primary, max)} + #{layout-width(secondary, max)} +
+  #{$layout-width-primary-max} + #{$layout-width-secondary-max} +
```

Comment:
Do we need an `@import` for `/styles/common'` here or is it okay since there's an import in `polaris-react/src/components/TopBar/TopBar.scss`?

Reply:
https://github.com/Shopify/polaris/pull/5722#discussion_r867244666